.github/workflows/testing.yml

```diff
@@ -24,14 +24,15 @@ jobs:
           version: v1.42.1
 
       - name: Run General Linting
-        uses: docker://ghcr.io/github/super-linter:slim-v4
+        uses: docker://ghcr.io/github/super-linter:slim-v4.8.5
         env:
           VALIDATE_ALL_CODEBASE: false
           VALIDATE_GO: false
           VALIDATE_JSCPD: false
           VALIDATE_GITHUB_ACTIONS: false
           VALIDATE_NATURAL_LANGUAGE: false
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VALIDATE_KUBERNETES_KUBEVAL: false
```

Comment:
Probably we should switch from opt-out model to opt-in and list all the things we want to lint explicitly. Btw - why do we have `VALIDATE_GO: false` if this is the main project language? =) Anyway, should be done in own PR

Reply:
`VALIDATE_GO: false` is because we use golangci-lint-action for linting go. And after that we use superlinter for all the rest.

Reply:
`super-linter` uses `golangci`under the hood, so it makes sense to use the super one